src/common/utilities/Utils.ts

```diff
@@ -285,6 +286,7 @@ export async function fetchRelatedFiles(activeFileUri: vscode.Uri, componentType
     } catch (error) {
         const message = (error as Error)?.message;
         telemetry.sendTelemetryErrorEvent(COPILOT_RELATED_FILES_FETCH_FAILED, { error: message, sessionId: sessionId });
+        oneDSLoggerWrapper.getLogger().traceError(COPILOT_RELATED_FILES_FETCH_FAILED, message, error as Error, { sessionId:sessionId }, {});
```

Comment:
is this a good naming convention for copilot events?

Reply:
updated